dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.39/V2_39_24__Create_table_userrolerestrictions.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE userrolerestrictions
```

Comment:
Should we do a drop table before we try to create it?

Reply:
`create table if not exists`